React/Modules/RCTDeviceInfo.m

```diff
@@ -153,6 +168,31 @@ - (void)_interfaceOrientationDidChange
   _currentInterfaceOrientation = nextOrientation;
 }
 
+
+- (void)interfaceFrameDidChange
+{
+  __weak typeof(self) weakSelf = self;
+  RCTExecuteOnMainQueue(^{
+    [weakSelf _interfaceFrameDidChange];
+  });
+}
+
+
+- (void)_interfaceFrameDidChange
+{
+  NSDictionary *nextInterfaceDimensions = RCTExportedDimensions(_bridge);
+
+  if (!([nextInterfaceDimensions isEqual:_currentInterfaceDimensions])) {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+      [_bridge.eventDispatcher sendDeviceEventWithName:@"didUpdateDimensions"
+                                                  body:RCTExportedDimensions(_bridge)];
```

Comment:
It would be good to not call `RCTExportedDimensions` two times, right?

Reply:
Great point, I'll update. Thanks!